### DISCUSSION TITLE
How did this user make a new branch in the repository?

### DISCUSSION THREAD

**Discussion by cclauss** [TheAlgorithms/Python#8573] [Team Posts]
@dhruvmanila Referring to https://github.com/TheAlgorithms/Python/pull/3514#issuecomment-712092926, let's have that conversation here.

**Comment by l3str4nge** [comment]
I don't get it. It seems like branch was created from my requested changes?

**Comment by dhruvmanila** [comment]
Only owners can check with the core settings of this repository. If possible, please alert them and ask them to check the settings.

**Comment by cclauss** [comment]
(I always wonder who the real OWNERS are. ;-)
